.github/workflows/enterprise-pr-check.yml

```diff
@@ -0,0 +1,18 @@
+name: Enterprise PR check
+on:
+  pull_request:
+    branches:
+      - main
+      - 'v[0-9]+.[0-9]+.x'
```

Comment:
Not sure about this syntax and how to properly test it.

Reply:
LGTM, we can test it in action.